.gjslintrc

```diff
@@ -0,0 +1,3 @@
+--max_line_length=120
+--exclude_directories=3rdparty
+--disable=0121,0220
```

Comment:
[The 16.04 package](http://packages.ubuntu.com/xenial/closure-linter) doesn't even check for this rule. I only had to turn it off for [the 14.04 package](http://packages.ubuntu.com/trusty/closure-linter).